src/module/scene/token-document/sheets/mixin.ts

```diff
@@ -26,11 +26,13 @@ function TokenConfigMixinPF2e<TBase extends ReturnType<typeof TokenApplicationMi
             },
         };
 
-        static override PARTS = (() => {
-            const parts = { ...super.PARTS };
-            parts["appearance"].template = "systems/pf2e/templates/scene/token/appearance.hbs";
-            return parts;
-        })();
+        static override PARTS = {
+            ...super.PARTS,
+            appearance: {
+                scrollable: [""],
+                template: "systems/pf2e/templates/scene/token/appearance.hbs",
+            },
+        };
```

Comment:
?

Reply:
That wasn't supposed to be part of this PR, it is from another change elsewhere, I'm sorry.
I'll update my fork so these don't happen.

Reply:
I know you did this locally to get your stuff to work, but I updated world explorer to handle scene config overrides better, since I ran into the same issue for Daggerheart. Since your OG implementation was world explorer's, I figure you should know.